src/main/java/org/micks/champmaker/championships/ChampionshipController.java

```diff
@@ -71,4 +76,9 @@ public void editingChampionship(@PathVariable long champId, @RequestBody Champio
     public void shuffleTeams(@PathVariable long champId) {
         championshipService.shuffleTeams(champId);
     }
+
+    @PostMapping(value = "/championships/{champId}/meals", consumes = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
We don't need `/championships` - we have that already on class level (my bad, forgot about that, when we discuss).